tox.ini

```diff
@@ -10,7 +10,12 @@ commands =
     coverage combine
     coverage report
     coverage xml
-passenv = DATABASE_URL PORT aws_access_key_id aws_secret_access_key
+passenv = 
+    DATABASE_URL PORT 
+    aws_access_key_id 
+    aws_secret_access_key 
+    is_aws_mturk_requestor 
+    mturk_worker_id
```

Comment:
I'm a little hesitant to "pollute" the `.dallingerconfig` file (and the config ecosystem in general) with settings used only in tests, but using environment variables for this purpose is tricky to document, if the expectation is that the tester is setting the values manually. 

What about a `.testconfig` file, which would be read by test setup, and if values were set in that file (it would have empty values by default), they would be set as environment variables?